pycbc/mchirp_area.py

```diff
@@ -21,20 +21,21 @@ def insert_args(parser):
                                              "source probabilities of a "
                                              "candidate event using the snr, "
                                              "mchirp, and effective distance.")
-    mchirp_group.add_argument('--src-class-mass-range', type=float, nargs=2,
-                              metavar=('MIN_M2', 'MAX_M1'),
-                              default=[1.0, 45.0],
+    mchirp_group.add_argument('--src-class-mass-ranges', type=float, nargs=3,
+                              metavar=('MIN_M2', 'MAX_M1', 'MAX_NS'),
+                              default=[1.0, 3.0, 45.0],
```

Comment:
Should it be clear which order these must be supplied in? Here you have max ns last in the list of metavars but the default has it second. Maybe it would be better to not try and rely on the numbers being in any order and instead sort them automatically and then assign the middle value as max ns?

Reply:
I put the metavars in that order because of the help text, but I think your option of sort them before it's better